test/bats/azure.bats

```diff
@@ -59,7 +61,7 @@ setup() {
 }
 
 @test "secretproviderclasses crd is established" {
-  cmd="kubectl wait --for condition=established --timeout=60s crd/secretproviderclasses.secrets-store.csi.x-k8s.io"
+  kubectl wait --for condition=established --timeout=60s crd/secretproviderclasses.secrets-store.csi.x-k8s.io
   wait_for_process $WAIT_TIME $SLEEP_TIME "$cmd"
```

Comment:
i think this line should be deleted unless you set `cmd`

Reply:
You're right! Deleted the line.